nuxeo-importer-core/src/main/java/org/nuxeo/ecm/platform/importer/base/GenericMultiThreadedImporter.java

```diff
@@ -346,7 +349,9 @@ protected void doRun() throws IOException {
         }
         stopImportProcrocess();
         log.info("All Threads terminated");
-        perfLogger.release();
+        if (enablePerfLogging && perfLogger != null) {
```

Comment:
It's enough to do `if (enablePerfLogging)` (same as when `perfLogger.log` is called higher).